src/Components/Timer/Timer.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import Timer from 'react-compound-timer';
```

Comment:
So cool that React has this built in as a package

Reply:
@sienna-kopf No kidding! Yay for resources!

Reply:
Yes!!! @Abdeboskey showed us this one from an Exquisite Corpse project he built last mod!

Reply:
I'm always impressed by its apparent simplicity!!